homeassistant/helpers/config_validation.py

```diff
@@ -219,6 +221,17 @@ def validate(obj):
     return validate
 
 
+def validate_values(valid_values):
```

Comment:
voluptuous has an `In` function that does exactly this. You call it as `vol.In([list, of, values])`.